replicatedhq/ship
# add in plan-confirm UI integration

What I Did
------------

Add code for presenting a plan to the user and waiting for them to confirm it


How I Did it
------------

- add new `tfplan` package that communicates with the daemon.Daemon
- add a step to `lifecycle.v1.terraform` that passes a fake plan to the user, waits for confirm/skip (both just do a skip for now)


How to verify it
------------

Run example, see plan, confirm, proceed.


Description for the Changelog
------------

Infrastructure steps now present a plan to the user.



![](https://upload.wikimedia.org/wikipedia/commons/f/fe/US_Navy_coastal_command_boat_on_crane_at_Bahrain_in_2014.JPG)
